lib/advisor_web/templates/provide_advice/advice-form.html.eex

```diff
@@ -4,6 +4,17 @@
       <h1>Advice for <%= @requester.name %></h1>
     </header>
 
+    <%= if @message do %>
+      <div class="message-area">
+        <p>A little note from <%= @requester.name %>:</p>
+        <blockquote class="message">
+          <%= @message %>
+        </blockquote>
+      </div>
+
+      <hr class="separator" />
```

Comment:
I would rather create a style that I apply on the previous div than using an `hr`

Reply:
We have used the `separators` across the entire app :)